src/stores/tab.ts

```diff
@@ -0,0 +1,6 @@
+import { atom } from 'recoil';
+
+export const mainTabState = atom<string>({
+  key: 'tab',
```

Comment:
selectedTabState로 변경하는 것은 어떨까요? 
key값은 아톰명의 'State'를 제외한 값으로 사용하기로 했어요!
```suggestion
export const selectedTabState = atom<string>({
  key: 'selectedTab',
```